### PR TITLE
XW-2587 | Portable Infoboxes: fixing image tabs

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-2587

//cc @Wikia/x-wing

### CODE REVIEW

**Review by kvas-damian — APPROVED** [review]

File: extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php [hunk 1]
```diff
@@ -145,6 +145,7 @@ protected function renderImage( $data ) {
 		$images = [ ];
 
 		for ( $i = 0; $i < count( $data ); $i++ ) {
+			$data[$i]['context'] = null;
```
Comment (latata) [inline extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php:148]: Wouldn't it be better to set it inside `extendImageData` function?

File: extensions/wikia/ArticleAsJson/ArticleAsJson.class.php [hunk 2]
```diff
@@ -257,7 +257,7 @@ public static function onExtendPortableInfoboxImageData( $data, &$ref ) {
 		wfProfileIn( __METHOD__ );
 
 		$title = Title::newFromText( $data['name'] );
-		if ( $title && F::app()->checkSkin( 'wikiamobile' ) ) {
+		if ( $title ) {
```
Comment (latata) [inline extensions/wikia/ArticleAsJson/ArticleAsJson.class.php:260]: I thought that ArticleAsJson class is only for mobile usage isn't it?

**Review by latata — APPROVED** [review]
LGTM

File: extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php (hunk 1 above)
Comment (rybmat) [reply extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php:148]: I don't think so, I think each of render service should set its context

**Pull request merged**